src/main/java/org/apache/mesos/specification/DefaultPhaseSpecification.java

```diff
@@ -0,0 +1,37 @@
+package org.apache.mesos.specification;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class provides a default implementation of the PhaseSpecification interface.
+ */
+public class DefaultPhaseSpecification implements PhaseSpecification {
+    private final String name;
+    private final List<TaskSpecification> taskSpecifications;
+
+    public DefaultPhaseSpecification(String name, TaskTypeSpecification taskTypeSpecification) {
```

Comment:
Given that the only caller of this constructor (in `DefaultPlanSpecificationFactory`) is getting the `name` param by just calling `taskTypeSpecification.getName()`, maybe remove that param and move the `getName()` call here inside the constructor?